aten/src/ATen/cuda/llvm_complex.cpp

```diff
@@ -485,6 +485,24 @@ operator&&(const complex<_Tp>& __x, const complex<_Tp>& __y)
     return (__x.real() || __x.imag()) && (__y.real() || __y.imag());
 }
 
+template<class _Tp>
+inline constexpr
+bool
+operator||(const complex<_Tp>& __x, const complex<_Tp>& __y)
+{
+    return (__x.real() || __x.imag()) || (__y.real() || __y.imag());
+}
+
+// logical_xor
+
+template<class _Tp>
+inline constexpr
+bool
+logical_xor(const complex<_Tp>& __x, const complex<_Tp>& __y)
```

Comment:
Looks like this is not used??

Reply:
Oops yes 😅 made corrections. Let's see if CI turns green ...

Reply:
This shouldn't be part of `llvm_complex.cpp` as it is not a standard library function (otherwise it will be include with other calls to jiterator with complex dtypes). I suggest to include this in logical_xor_string.

Reply:
Done, removed from there :)